Example/Backpack.xcodeproj/project.pbxproj

```diff
@@ -1186,7 +1193,7 @@
 				CLANG_WARN_ASSIGN_ENUM = YES;
 				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = YES;
 				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
-				DEVELOPMENT_TEAM = WZLXWEEH2U;
+				DEVELOPMENT_TEAM = "";
```

Comment:
This shouldn't have changed...

Reply:
I think this should have? We don't have a development team for this app so I think it should be blank

Reply:
Fair enough 👍 
It shouldn't change in the future then 😛